app/MMB/src/scenes/tripOverview/TripOverviewTablet.js

```diff
@@ -0,0 +1,31 @@
+// @flow strict
+
+import * as React from 'react';
+import { graphql, PrivateApiRenderer } from '@kiwicom/mobile-relay';
+
+import TripOverview from './TripOverview';
+import BookingDetailContext from '../../context/BookingDetailContext';
+
+function TripOverviewTablet(rendererProps) {
```

Comment:
missing flow types

Reply:
Seems like Flow is clever enough to understand what's going on here (it would complain otherwise). Fixed... 🙂